resources/js/Pages/Collections/Components/CollectionsFilter/CollectionsFilterPopover.tsx

```diff
@@ -38,6 +45,13 @@ export const CollectionsFilterPopover = ({
         }
     }, [debouncedQuery]);
 
+    const isNetowrkFilterActive =
+        isLoading ||
+        availableNetworks
+            .filter((network) => network.collectionsCount > 0)
+            .every((network) => selectedChainIds.includes(network.chainId)) ||
+        selectedChainIds.length === 0;
```

Comment:
logic is still flawed when you only have one of the two networks available, as it will show the notification dot when "all" are deselected now

<img width="354" alt="image" src="https://github.com/ArdentHQ/dashbrd/assets/35610748/51f4adf9-bc58-4d82-b410-c0ab260014a0">